pubnub/pubnub.py

```diff
@@ -101,6 +101,13 @@ def __init__(self, pubnub):
     def _register_heartbeat_timer(self):
         self.stop_heartbeat_timer()
 
+        if self._retry_limit_reached():
```

Comment:
I don't know how it works in Python, but should announcing `disconnected` status being part of EE?

Reply:
No. This one is for non-event engine subscribe (yeah, python has more than one subscribe)